extensions/sql-database-projects/src/common/azureFunctionsUtils.ts

```diff
@@ -0,0 +1,73 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as utils from './utils';
+import * as constants from './constants';
+import { parseJson } from './parseJson';
+
+/**
+ * Represents the settings in an Azure function project's local.settings.json file
+ */
+export interface ILocalSettingsJson {
+	IsEncrypted?: boolean;
+	Values?: { [key: string]: string };
+	Host?: { [key: string]: string };
+	ConnectionStrings?: { [key: string]: string };
+}
+
+/**
+ * copied and modified from vscode-azurefunctions extension
+ * @param localSettingsPath full path to local.settings.json
+ * @returns settings in local.settings.json
+ */
+export async function getLocalSettingsJson(localSettingsPath: string): Promise<ILocalSettingsJson> {
+	if (await fse.pathExists(localSettingsPath)) {
+		const data: string = (await fse.readFile(localSettingsPath)).toString();
+		if (/[^\s]/.test(data)) {
+			try {
+				return parseJson(data);
+			} catch (error) {
+				const message: string = constants.localize('failedToParse', 'Failed to parse "{0}": {1}.', constants.azureFunctionLocalSettingsFileName, error.message);
```

Comment:
I was thinking of not changing the copied over code that much so it's easier to compare in the future if needed, but I'll move these strings since those should be too confusing